AllReadyApp/Web-App/AllReady.UnitTest/Features/Notifications/NotifyVolunteerForUserUnenrollsTests.cs

```diff
@@ -16,543 +16,336 @@ namespace AllReady.UnitTest.Features.Notifications
-            NotifyVolunteersCommand command = null;
+            var userId = "user1@example.com";
+            var userEmail = "useremail1@example.com";
+            
+            var notification = new UserUnenrolls { UserId = userId };
 
-            var notification = GetNotification(userId);
-            var model = GetEventDetailForNotificationModel();
+            var model = new EventDetailForNotificationModel
+            {
+                UsersSignedUp = new List<EventSignup> { new EventSignup { User = new ApplicationUser { Id = userId, Email = userEmail } } }
+            };
 
-            model.UsersSignedUp.RemoveAll(u => u.User.Id == model.Volunteer.Id);
+            var mockMediator = new Mock<IMediator>();
+            mockMediator.Setup(x => x.SendAsync(It.IsAny<EventDetailForNotificationQueryAsync>())).ReturnsAsync(model);
 
-            _mediator.Setup(m => m.SendAsync(It.IsAny<EventDetailForNotificationQueryAsync>())).ReturnsAsync(model);
-            _mediator.Setup(m => m.SendAsync(It.IsAny<NotifyVolunteersCommand>()))
-                .Returns((NotifyVolunteersCommand c) =>
-                {
-                    command = c;
-                    return Task.FromResult(new Unit());
-                });
+            var options = new Mock<IOptions<GeneralSettings>>();
+            options.Setup(o => o.Value).Returns(new GeneralSettings());
 
-            await _handler.Handle(notification);
+            var handler = new NotifyVolunteerForUserUnenrolls(mockMediator.Object, options.Object);
+            await handler.Handle(notification);
 
-            command.ShouldBeNull();
+            mockMediator.Verify(x => x.SendAsync(It.Is<NotifyVolunteersCommand>(c =>
+                c.ViewModel.EmailRecipients != null
+                && c.ViewModel.EmailRecipients.Count == 1
+                && c.ViewModel.EmailRecipients.First() == userEmail
+            )), Times.Once);
         }
 
-        //[Fact]
-        //public void NotifyVolunteerForUserUnenrolls_NotSent_EventIsNull()
-        //{
-        //    var userId = "volunteer1@example.com";
-        //    NotifyVolunteersCommand command = null;
-
-        //    var notification = GetNotification(userId);
-        //    EventDetailForNotificationModel model = null;
-
-        //    _mediator.Setup(m => m.SendAsync(It.IsAny<EventDetailForNotificationQueryAsync>())).ReturnsAsync(model);
-        //    _mediator.Setup(m => m.SendAsync(It.IsAny<NotifyVolunteersCommand>()))
-        //        .Returns((NotifyVolunteersCommand c) => {
-        //            command = c;
-        //            return Task.FromResult(new Unit());
-        //        });
-
-        //    Should.Throw<NullReferenceException>(async() => await _handler.Handle(notification));            
-        //}
-
         [Fact]
-        public async Task NotifyVolunteerForUserUnenrolls_NotSent_EventIsEmpty()
+        public async Task SendNotifyVolunteersCommand_WithCorrectSiteBaseUrl()
```

Comment:
@mgmccarthy , given we are consolidating all the tests asserting different parts of the message into one asserting the complete message, do we still need to test eventLink separately? It's included in the message body.